features/taxation/managing_tax_rates/adding_tax_rate.feature

```diff
@@ -0,0 +1,24 @@
+@managing_tax_rates
+Feature: Adding a new tax rate
+    In order to apply different taxes rates to various tax categories
+    As an Administrator
+    I want to add a new tax rate to the registry
+
+    Background:
+        Given there is "EU" zone containing all members of European Union
+        And the store has a tax category "Food and Beverage" with a code "food_and_beverage"
```

Comment:
Can we get rid of the tax category code? It does not have any meaning in this feature.